core/kazoo_number_manager/src/knm_numbers.erl

```diff
@@ -262,20 +262,20 @@ do_get_pn(Nums, Options, Error) ->
 %%--------------------------------------------------------------------
 -spec create(ne_binaries(), knm_number_options:options()) -> ret().
 create(Nums, Options) ->
-    ?MATCH_ACCOUNT_RAW(AccountId) = knm_number_options:assign_to(Options), %%FIXME: can crash
```

Comment:
This ensures one cannot create numbers that would not be assigned to anyone. 
I realize now that this line is missing a test case.